tools/package-tools/version-placeholders.ts

```diff
@@ -0,0 +1,32 @@
+import {readFileSync, writeFileSync} from 'fs';
```

Comment:
Good question. Looks like the old one before the build changes is still there after rebasing.